Fundamentals 1/data-types.html/data-types.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Data Types</title>
+    <style>
+        h1, h3 {
+            text-align: center;
+        }    
+         h3 {
+             margin-top: 100px;
+         }    
+        table{
+            font-size: 20px;
+            border-collapse: collapse;
+            width: 50%;
+            margin: 0 auto;
+        }
+        th {
+            border: 1px solid #dddddd;
+            text-align: left;
+            padding: 8px;
+        }
+        tr:nth-child(even) {
+            background-color: #dddddd;
+        }
+
+        img {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+            width: 50%;
+        }
+        </style>
+</head>
+<body>
+    <h1>Basic Data Types</h1>
+    <table>
+        <tr>
+           <th> "Strings" : a series of characters within double quotes " " or single quotes '' . Strings cannot be changed only replaced.</th> 
+        </tr>
+        <tr>
+            <th>Numbers : with or without decimals. If you use a number within a double or a single quote, it will be treated as a string and not as a number. </th>
+        </tr>
+        <tr>
+            <th>BigInit : an integer with arbitrary precision.</th>
+        </tr>
+        <tr>
+            <th>Booleans have two values : true & false.</th>
+        </tr>
+        <tr>
+            <th>Undefined : a variable that a value hasn't been assigned.</th>
+        </tr>
+        <tr>
+            <th>Null : has only one value, null, or as in Python (that makes more sense) None.</th>
+        </tr>
+
+    </table>
+
+    <h2>Check out this <span><a href= "https://www.programiz.com/javascript/data-types" target="blank"> source </a></span></h2>
```

Comment:
What's the use of the span here?

Reply:
Never understood why I should use a span, so I wanted to use it. Also, I don't use div, still don't know why.

Reply:
We try to avoid div and instead use the Semantic HTML elements. Spans are used whenever we want to target some characters or text inside another element: <p>The letter <span class="color_me_red">A</span> will be red</p>